### DISCUSSION TITLE
External IP addresses during installation and use

### DISCUSSION THREAD

**Discussion by woodroom** [nocodb/nocodb#788] [Discord Threads]
Due to the peculiarities of the information security policy in my company, I need a complete list of addresses to which the service accesses
1) during installation
2) during operation
While installing, I see the request to https://registry.npmjs.org.
Installation does not proceed further, because the firewall has blocked access to any unauthorized addresses.
Please help.

**Comment by o1lab** [comment]
Curious : what industry your company belongs to ?

A node project cannot run without npm install and installing from  https://registry.npmjs.org. We check dockerhub api to see if a new version is available and telemetry.

Your best bet is to run docker container with telemetry disabled. That doesnt stop querying docker hub api though.

**Comment by o1lab** [comment]
Moved to discussion.
